.yamato/all-smoke_test.yml

```diff
@@ -30,10 +34,6 @@ Extra_Smoke_Test_trunk:
     dependencies:
       -  path: .yamato/all-smoke_test.yml#PR_Smoke_Test_trunk
```

Comment:
Is the dependency on the PR job intended? (I think this was from an earlier PR, not this one, but wondering it nevertheless)